Models/userModel.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('../DataBase/connection');
+const UserSchema = mongoose.Schema({
+   
+    name: String,
+    zip: Number,
```

Comment:
We should probably include street address for the user. Provider needs to get to their house

Reply:
Also should include a key value pair for email, and one for password as well